### PR TITLE
koordlet: support system config like minFreeKbytes,waterMarkScaleFactor

### PR DESCRIPTION
**Pull request by chzhj**
Signed-off-by: chzhj <blueheartcheng@gmail.com>

### Ⅰ. Describe what this PR does
support system config 
see this proposal for details: https://github.com/koordinator-sh/koordinator/issues/977

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Review by FillZpp — APPROVED** [review]
/lgtm

File: apis/extension/slo_controller_config.go [hunk 1]
```diff
@@ -210,6 +224,13 @@ data:
         }
       ]
     }
+  system-config: |-
```
Comment (jasonliu747) [inline apis/extension/slo_controller_config.go:227]: 需不需要加上`NodeStrategies`的例子？

**Review by jasonliu747 — APPROVED** [review]

File: apis/extension/slo_controller_config.go (hunk 1 above)
Comment (chzhj) [reply apis/extension/slo_controller_config.go:227]: OK，can add NodeStrategies in next pr https://github.com/koordinator-sh/koordinator/issues/1011

**Pull request merged**